input/parser.go

```diff
@@ -2,68 +2,80 @@ package input
 
 import (
 	"fmt"
-	"strings"
+	"github.com/ekanite/ekanite/parser"
+	"log"
 )
 
-var (
-	fmtsByStandard = []string{"rfc5424"}
-	fmtsByName     = []string{"syslog"}
+const (
+	Rfc5424Standard   = "RFC5424"
```

Comment:
Go naming convention dictates these enums should start "RFC".  See https://talks.golang.org/2014/names.slide#5